src/.vuepress/theme/Layout.vue

```diff
@@ -1,13 +1,13 @@
 <template>
   <div class="md-layout">
     <div class="overlay" :class="{hidden: !sidebarOpen}" @click="closeSidebar"></div>
-    <Header ref="header" @sidebar-open="openSidebar"/>
+    <Header ref="header" @sidebar-open="openSidebar" :links="links"/>
     <div ref="container" class="md-container">
       <!-- Main container -->
       <main class="md-main">
         <div class="md-main__inner md-grid" data-md-component="container">
           <!-- Main navigation -->
-          <Sidebar ref="sidebar" v-if="!$page.frontmatter.nosidebar" :sidebar-open="sidebarOpen"/>
+          <Sidebar ref="sidebar" v-if="!$page.frontmatter.nosidebar" :sidebar-open="sidebarOpen" @closeSidebar="closeSidebar" :links="links"/>
```

Comment:
Events should either be in camelCase or in kebab-case, either (verb, noun) or (noun, verb). Currently we have `sidebar-open` (noun-verb) and `closeSidebar` (verbNoun).
Feel free to harmonize as you feel.

Reply:
Even if sidebarOpen is a variable and closeSidebar an event ? 
I was thinking that's more explicit like this because we will read the sidebar(is)Open value et we'll call closeSidebar ... I'll do the change !